application/src/main/java/org/opentripplanner/standalone/configure/ConstructApplicationModule.java

```diff
@@ -78,6 +78,7 @@ OtpServerRequestContext providesServerContext(
       streetLimitationParametersService,
       transitRoutingConfig,
       transitService,
+      routerConfig.triasApiConfig(),
```

Comment:
Follow the existing convention with a member variable.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/6510/commits/0edfd73ea6cf3a8d0d847525aa84f4429394aead